alvr/graphics/Cargo.toml

```diff
@@ -9,6 +9,7 @@ license.workspace = true
 [dependencies]
 alvr_common.workspace = true
 alvr_session.workspace = true
+alvr_packets.workspace = true
```

Comment:
Tiny nit: put dependencies in alphabetical order (but alvr_* still hoisted).

But actually i think what we should do is move ViewParams to alvr_common

Reply:
Resolved, removed